### PR TITLE
Fix build failure

### PR DESCRIPTION
**Pull request by JuanMaRuiz**
Although the coverage of psi is great

test-coverage

I think there are some minor changes that could be included in order to have more stable tests.

In the last PR we had some problems in the test, marking the build as instable, due a connection with the API. Another problem, related with travis and web requests was solved in the pass.

If we take a look to [`api-response.test.js`](https://github.com/GoogleChromeLabs/psi/blob/master/test/api-response.test.js) we see that tests are:

* Connecting with the real API.
* Checking the response from the API.

From my point of view we should avoid this call to the real API and test should check the methods instead.

This PR contains several changes that try to solve that.

Hope it helps.

Best regards.

### CODE REVIEW

File: test/api-response.test.js [hunk 1]
```diff
@@ -1,20 +0,0 @@
-/* eslint-env mocha */
```
Comment (exterkamp) [inline test/api-response.test.js]: This should be g2g now, there was a systemic issue in PSI recently summarized here https://github.com/GoogleChrome/lighthouse/issues/9902#issuecomment-556414195

I think we can leave this test :smile:

File: test/index.test.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    };
+
+    it('should return strategy "mobile" if no strategy is passed', () => {
+      const expectedOutput = {
+        ...defaultConfig,
+        strategy: 'mobile'
+      };
+      expect(handleOptions('http://addyosmani.com')).to.eql(expectedOutput);
+    });
+    it('should return strategy "desktop" if "desktop" is passed as strategy', () => {
```
Comment (exterkamp) [inline test/index.test.js]:
```suggestion
    it('should respect passed strategy options', () => {
```

File: test/output.test.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+const outputWired = rewire('./../lib/output.js');
+const getThreshold = outputWired.__get__('getThreshold');
+
+describe('getThreshold method', () => {
+  it('should return "70" if no valid value is passed', () => {
+    expect(getThreshold('string')).to.equal(70);
+    expect(getThreshold(false)).to.equal(70);
+    expect(getThreshold(101)).to.equal(70);
+  });
+  it('should return the passed threshold value if correct', () => {
```
Comment (exterkamp) [inline test/output.test.js]:
```suggestion
  it('should respect passed threshold values', () => {
```

File: test/api-response.test.js (hunk 1 above)
Comment (JuanMaRuiz) [reply test/api-response.test.js]: Hi @exterkamp many thanks for your review. 

Much better your proposal for the test definitions 😃.

I have added `api-response.test.js` again.

I have also reverted some changes that I mixed up 🤦‍♂️ (.gitignore and package.json). Sorry about that.

Hope now everything is working as expected.

Best regards.

**Comment by JuanMaRuiz** [comment]
No problem, I think is a better solution. The initial idea was not changing the original code too much, but I prefer your solution.

Best regards.

File: lib/output.js [hunk 4]
```diff
@@ -91,8 +90,6 @@ function getReporter(format) {
 
 const convertToPercentum = num => num * 100;
 
-const getThreshold = threshold => (typeof threshold === 'number' && threshold && threshold < 100) ? threshold : THRESHOLD;
```
Comment (exterkamp) [inline lib/output.js]: Since we're pulling this function out, does it make sense to pull out `getReporter()` as well? It is doing the same sort of thing.

Comment (JuanMaRuiz) [reply lib/output.js]: Of course, no problem. I'll land this.

Comment (JuanMaRuiz) [reply lib/output.js]: Hi @exterkamp this request has been done.

**Comment by exterkamp** [comment]
Pro

:pray: image :pray:

**Pull request merged**